test/eventing-kafka.bash

```diff
@@ -29,3 +29,29 @@ function upstream_knative_eventing_kafka_e2e {
   logger.info 'Starting eventing-kafka tests'
   run_e2e_tests
 }
+
+function upstream_knative_eventing_kafka_broker_e2e {
+  logger.info 'Setting Kafka as default broker class'
+
+  local root_dir
+  root_dir=$(dirname "$0")/..
```

Comment:
Are you sure this is the right ?

Reply:
I think this should be:
```
rootdir=""$(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")"
```

Reply:
```suggestion
  root_dir="$(dirname "$(dirname "$(realpath "${BASH_SOURCE[0]}")")")"
```

Reply:
That looks beautiful :joy:

Reply:
Right ?!? :smile:  But it should be bullet-proof cos it doesn't depend on where you're running the tests from.